### ISSUE TITLE
[a11y] 2.4.3 Conclusive sequence for keyboard operation

### ISSUE THREAD

**Issue by tschiebel** [owncloud/client#11586]
https://infinite.owncloud.com/s/KcGPhbRgJfHYHuw

[BIT inklusiv BITV-Softwaretest - BIT inklusiv (bit-inklusiv.de)](https://www.bit-inklusiv.de/bitv-softwaretest/)

### Space, Server activity and Synchronization protocol

- [x] The elements of the navigation (toolbar) at the top of the page receive the keyboard focus 
in the wrong order, which is quite confusing.
![image](https://github.com/owncloud/client/assets/132444655/ebf82348-a9e4-4583-9e3d-773847917965)

### Settings

- [x] The "Language" drop-down element does not receive the keyboard focus 
according to its visual arrangement as part of the "General settings" group 
Settings" group, but after the elements of the "Advanced - Approve synchronization of folders
Approve synchronization of folders ..." group.

- [x] The elements of the navigation (toolbar) at the top of the page receive the 
keyboard focus in reverse order, which is quite confusing.

### Add Space 

- [x] The data cells of the table columns "Image" and "Name" can be focused with the keyboard 
which is not expected.
